package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "agroeasy",
-  "version": "1.1.0",
+  "version": "1.1.2",
```

Comment:
Why did you change the version from 1.1.1 in the previous PR to 1.1.2?

Reply:
So this is called [semantic versioning](https://semver.org/). It lets developers know that a change has been made to the code base. The changes that I am making right now are typed to version 1.1.2. Next week, I will introduce semantic version to you all. It is incredibly important but simple to understand.

Reply:
@Simpcyclassy good job in spotting this

Reply:
OK. That's cool

Reply:
thats my girl